package/src/main/java/com/microsoft/azure/cosmos/cassandra/CosmosRetryPolicy.java

```diff
@@ -43,13 +45,14 @@
 public class CosmosRetryPolicy implements RetryPolicy {
 
     public CosmosRetryPolicy(int maxRetryCount) {
```

Comment:
Instead of updating the existing constructor, we should add a new default Constuctor which defaults to useRetryMillisIfAvailable=true